src/main/java/MainOptions.java

```diff
@@ -0,0 +1,80 @@
+import java.util.Scanner;
+
+public class MainOptions {
```

Comment:
O que acha de chamar de Menu?
```suggestion
public class Menu {
```